phost/src/main.rs

```diff
@@ -451,11 +462,11 @@ async fn update_singer_nonce(client: &XtClient, signer: &mut subxt::PairSigner<R
     Ok(())
 }
 
-async fn sync_tx_to_chain(client: &XtClient, pr: &PrClient, sequence: &mut u64, pair: sr25519::Pair) -> Result<(), Error> {
+async fn sync_tx_to_chain(client: &XtClient, pr: &PrClient, mut sequence: &mut u64, pair: sr25519::Pair, contract_id: u32) -> Result<(), Error> {
```

Comment:
Do we need `mut sequence` here? Looks like we can keep `sequence: &mut u64`. And at the bottom:

```rust
submit_balance/asset_transactions(&client, sequence, pair, transfer_data).await
```